code/modules/cargo/packs/service.dm

```diff
@@ -280,6 +280,15 @@
 	crate_type = /obj/structure/closet/crate/large
 	discountable = SUPPLY_PACK_UNCOMMON_DISCOUNTABLE
 
+/datum/supply_pack/service/fish_feed
```

Comment:
Make this a goodie, like almost every fishing-related pack.

Reply:
Wait really? Man, I was gone for too long.

Reply:
Ok, now that the goodies pack has been added, can you remove this one? I don't think it's a good idea to have two of the same pack in different categories, also people will mostly ignore this one.